.gitignore

```diff
@@ -5,6 +5,8 @@ capybara-*.html
 /tmp
 /db/*.sqlite3
 /db/*.sqlite3-journal
+/db/postgres
```

Comment:
How are you getting these files here though? aren't they in you system postgres directory?